Core/include/Acts/Seeding/BinnedSPGroup.ipp

```diff
@@ -30,7 +34,7 @@ Acts::BinnedSPGroupIterator<external_spacepoint_t>::operator++() {
   // if we were on the edge, go up one phi bin and reset z bin
   if (++m_current_localBins[INDEX::Z] == m_max_localBins[INDEX::Z]) {
     ++m_current_localBins[INDEX::PHI];
-    m_current_localBins[INDEX::Z] = 0;
+    m_current_localBins[INDEX::Z] = m_skipZMiddleBin;
```

Comment:
what does m_skipZMiddleBin actually mean? we have the custom looping on the z axis. Is `m_skipZMiddleBin ` saying to skip the first N elements in the grid, or the first N elements in the custom looping?

for ITk pixel we have `zBinsCustomLooping = [1,2,3,4,11,10,9,8,6,5,7]` . If `m_skipZMiddleBin` is for instance 3, what bin will we skip?

Reply:
`m_skipZMiddleBin` is used to skip the first N elements in the custom looping. 

For the fast tracking we have:
`zBinsCustomLooping = [1, 11, 2, 10, 3, 9, 6, 4, 8, 5, 7]` and `skipZMiddleBinSearch = 2`
so we skip 1 and 11 which are the outer z bins (we are basically skipping outer end-cap layers)

if `skipZMiddleBinSearch` were 7 we would skip the first 7 bins in z according to zBinsCustomLooping